Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.3-slim
+FROM ruby:2.7
```

Comment:
I ended up adding the packages to compile bigdecimal gem.  Not sure this would help.

```
RUN apt-get update -y && \
    apt-get install -y --no-install-recommends \
      build-essential \
      git \
      libffi-dev \
      gcc \
      make
```

Reply:
It does, but there's not much point when the non-slim image contains the required packages pre-installed.